templates/my-orders.php

```diff
@@ -54,13 +90,20 @@
                         <td class="order-total">
                             <?php echo wp_kses_post( sprintf( _n( '%s for %s item', '%s for %s items', $item_count, 'dokan-lite' ), $order->get_formatted_order_total(), $item_count ) ); ?>
                         </td>
+                        <td class="order-total">
+                            <?php
+                            foreach ( $order->get_items() as $id => $product ) {
```

Comment:
Use `foreach : endforeach;` notation.
Also, as `$id` isn't used, it should not be declared.